### ISSUE TITLE
Difference between code and description in the paper

### ISSUE THREAD

**Issue by ajtejankar** [om-ai-lab/VL-CheckList#10]
Hi,

Thanks for open sourcing your code. I am trying to reproduce the results for ALBEF in your paper, but no success. I was going through your code and noticed that ITM logits/probabilities are used differently in the code than in the paper. Paper describes, "If the model score on the original text description is higher than the score on the generated negative samples, we regard it as positive output." However, in the [code](https://github.com/om-ai-lab/VL-CheckList/blob/9e6b5ef13e1133bfa37d54748d368e46d80dfdb1/vl_checklist/evaluate.py#L55) only the ITM logit corresponding to  "matching" `z[1]` is used. Basically, the code never compares the scores between positive and negative text as described in the paper. Can you please clarify?

Thanks,
Ajinkya

**Comment by kyusonglee** [comment]
We set the self.task == 'itc' when evaluating the score in the paper.
